gwdetchar/io/html.py

```diff
@@ -247,6 +319,238 @@ def new_bootstrap_page(base=os.path.curdir, lang='en', refresh=False,
+    for i, link in enumerate(links):
+        if isinstance(active, int) and i == active:
+            active_ = True
+        elif isinstance(active, (list, tuple)) and i == active[0]:
+            active_ = active[1]
+        else:
+            active_ = False
+        dropdown_link(page, link, active=active_, class_=column)
+    page.ul.close()
+    return page()
+
+
+def dropdown_link(page, link, active=False, class_=''):
+    """Format links within a dropdown menu
+
+    Parameters
+    ----------
+    page : `~MarkupPy.markup.page`
+        a `page` object to format in-place
+
+    link : `~MarkupPy.markup.page` or `list`
+        the link(s) to format
+
+    active : `bool`, optional
+        boolean switch to enable (`True`) or disable (`False`) an active link,
+        default: `False`
+
+    class_ : `str`, optional
+        object class of the link, default: `''`
+    """
+    if link is None:
+        page.li(class_='divider')
+    elif active is True:
+        page.li(class_='active')
+    else:
+        page.li()
+    if isinstance(link, (tuple, list)):
+        if isinstance(link[1], (tuple, list)):
+            page.ul(class_=class_ + ' list-unstyled')
+            page.li(link[0], class_='dropdown-header')
+            for j, link2 in enumerate(link[1]):
+                dropdown_link(page, link2,
+                              active=(type(active) is int and active == j))
+            page.ul.close()
+        else:
+            page.a(link[0], href=link[1])
+    elif link is not None:
+        page.add(str(link))
+    page.li.close()
+
+
+def get_brand(ifo, name, gps, about=None):
+    """Return a brand for navigation bar formatting
+
+    Parameters
+    ----------
+    ifo : `str`
+        interferometer prefix for color-coding, e.g. `'L1'`
+
+    name : `str`
+        name of the analysis, e.g. `'Scattering'`
+
+    gps : `float`
+        GPS second of the analysis
+
+    about : `str`, optional
+        relative path to the `about` page for this analysis, default: None
+
+    Returns
+    -------
+    brand : `~MarkupPy.markup.page`
+        the navbar brand `page` object
+
+    class_ : `str`
+        object class of the navbar
+    """
+    page = markup.page()
+    page.div(ifo, class_='navbar-brand')
+    page.div(name, class_='navbar-brand')
+    page.div(class_='btn-group pull-right ifo-links')
+    page.a(class_='navbar-brand dropdown-toggle', href='#',
+           **{'data-toggle': 'dropdown'})
+    page.add('Links')
+    page.b('', class_='caret')
+    page.a.close()
+    page.ul(class_='dropdown-menu')
+    if about is not None:
+        page.li('Internal', class_='dropdown-header')
+        page.li()
+        page.a('About this page', href=about)
+        page.li.close()
+        page.li('', class_='divider')
+    page.li('External', class_='dropdown-header')
+    for name, url in OBSERVATORY_MAP[ifo]['links'].items():
+        if 'Summary' in name:
+            day = str(tconvert(gps).date()).replace('-', '')
```

Comment:
This might better as
```suggestion
            day = from_gps(gps).strftime(r"%Y%m%d")
```